File: DemoApp-Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/SplashActivity.java
package com.applovin.enterprise.apps.demoapp;

import android.content.Intent;
import android.os.Bundle;
import android.text.Html;
import android.widget.TextView;

import com.applovin.sdk.AppLovinSdkUtils;

import java.util.concurrent.TimeUnit;

import androidx.appcompat.app.AppCompatActivity;

public class SplashActivity
        extends AppCompatActivity
{
    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate( savedInstanceState );
        setContentView( R.layout.activity_splash );

        TextView logoTextView = findViewById( R.id.logoTextView );
        logoTextView.setText( Html.fromHtml( "<b>App</b>Lovin" ) );

        AppLovinSdkUtils.runOnUiThreadDelayed( () -> {

            Intent intent = new Intent( SplashActivity.this, MainActivity.class );
            intent.addFlags( Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK | Intent.FLAG_ACTIVITY_NO_ANIMATION );
            startActivity( intent );
        }, TimeUnit.SECONDS.toMillis( 2 ) );
    }
}

File: DemoApp-Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/SplashActivity.kt
package com.applovin.enterprise.apps.demoapp

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.core.text.HtmlCompat
import com.applovin.enterprise.apps.demoapp.kotlin.R
import com.applovin.sdk.AppLovinSdkUtils
import kotlinx.android.synthetic.main.activity_splash.*
import java.util.concurrent.TimeUnit

class SplashActivity : AppCompatActivity()
{
    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        logoTextView.text = HtmlCompat.fromHtml("<b>App</b>Lovin", HtmlCompat.FROM_HTML_MODE_LEGACY)

        AppLovinSdkUtils.runOnUiThreadDelayed(
                {
                    val intent = Intent(this, MainActivity::class.java)
                    intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK or Intent.FLAG_ACTIVITY_NO_ANIMATION)
                    startActivity(intent)
                }, TimeUnit.SECONDS.toMillis(2))
    }
}
